front/src/views/Home.vue

```diff
@@ -32,26 +52,19 @@
                     </v-card>
                 </v-col>
 
-                <v-col cols="12" v-if="team">
-                    <v-card color="blue lighten-1" dark>
-                        <v-card-title class="headline">The Gif of the Day</v-card-title>
+                <v-col cols="12">
+                    <v-card color="blue-grey darken-1" dark>
+                        <v-card-title class="headline">Your team</v-card-title>
```

Comment:
Maybe replace Your team by FAQ ?

Reply:
oopsie it is a fail!

Reply:
resolved by https://github.com/ytvnr/gif-of-the-day/pull/120/commits/db8767f4e0a5da45463ad783f6915a8b74e6f372